packages/orm/src/EntityManager.ts

```diff
@@ -398,6 +398,73 @@ export class EntityManager<C = {}> {
     return createOrUpdatePartial(this, type, opts);
   }
 
+  /** Utility to clone an entity and its nested relations, as determined by a populate hint */
+  public async cloneEntity<T extends Entity, H extends LoadHint<T>>(entity: T, hint?: H): Promise<Loaded<T, H>> {
```

Comment:
thought: Since this will be used like `em.cloneEntity()`, I wonder if `em.clone()` would be better. Maybe just `clone` will make it more apparent you are going to clone everything on the hint, not just the provided `entity`?

Reply:
It might also be helpful to include a usage example!

```
/**
 * Utility to clone an entity and its nested relations, as determined by a populate hint
 *
 * @example
 * // This will duplicate the author and all book entities
 * const duplicateBook = await em.clone(author, "books"); 
 */
```

Following https://jsdoc.app/tags-example.html

Reply:
Yeah @chr1sjf0x I think that could be a good call